package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vite-plugin-cloudflare-redirect",
   "description": "Vite plugin to handle Cloudflare's _redirects file",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
I don't have permissions to update this branch, can your revert this part, as I'll update it later on release.